src/tests/test_campaign.py

```diff
@@ -41,14 +41,9 @@ def setUp(self):
 
     with self.app.app_context():
       # create all db objects
+      db.drop_all()
       db.create_all()
```

Comment:
`drop_all()` is the same as finding campaigns by name and then deleting them

Reply:
Nice!